flink-runtime/src/main/java/org/apache/flink/runtime/rest/handler/legacy/metrics/JobMetricsHandler.java

```diff
@@ -32,8 +46,11 @@
  * The handler will then return a list containing the values of the requested metrics.
  * {@code [ { "id" : "X", "value" : "S" }, { "id" : "Y", "value" : "T" } ] }
  */
-public class JobMetricsHandler extends AbstractMetricsHandler {
+public class JobMetricsHandler extends AbstractMetricsHandler
```

Comment:
Why aren't we simply implementing a new handler? I think that we don't reuse any of `AbstractMetricsHandler` functionality and, thus, there is no need to use it as a base class.